dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -420,7 +420,7 @@ protected boolean verifyMethodConfig(
     }
 
     private ArgumentConfig getArgumentByIndex(MethodConfig methodConfig, int argIndex) {
-        if (methodConfig.getArguments() != null && methodConfig.getArguments().size() > 0) {
+        if (methodConfig.getArguments() != null && !methodConfig.getArguments().isEmpty()) {
```

Comment:
**Optimization Issue**: Improvement by replacing 'methodConfig.getArguments().size() > 0' with '!methodConfig.getArguments().isEmpty()' for better readability and performance. <br> **Fix**: Utilize the isEmpty method for collection size checking to improve code readability and performance. <br> **Code Suggestion**: 
 ```
 -        if (methodConfig.getArguments() != null && methodConfig.getArguments().size() > 0) {
 +        if (methodConfig.getArguments() != null && !methodConfig.getArguments().isEmpty()) {
 ```